### ISSUE TITLE
Strange blending / alpha issue in -qt version vs. chrome.

### ISSUE THREAD

**Issue by Sloanstar** [moonlight-stream/moonlight-qt#32]
There's a strange alpha blend / dithering effect / encoding effect that goes across the screen when compared to chrome version.

All -qt versions released up to and including v0.1.0

GFE 3.14.1.48
GF Drivers 398.82
Win10 1803 (x64)

Bitrate 150Mbps
1080p
60 FPS
All other moonlight client settings Automatic/Autodetect/Defaults

![2018-08-09 20_47_12-moonlight_copy](https://user-images.githubusercontent.com/16750081/43933234-7bd82828-9c16-11e8-96c6-f001d2d68ba6.png)

![inked2018-08-09 20_33_21-moonlight_li](https://user-images.githubusercontent.com/16750081/43933244-988223d4-9c16-11e8-987b-2352c2f0848e.jpg)

**Comment by cgutman** [comment]
Can you attach the Moonlight log file? On Windows, it's in %TEMP% and will be called `Moonlight-<some numbers>.log`.

After getting the log of your current configuration, do you see the same issue if you force software decoding in the settings?

**Comment by Sloanstar** [comment]
[Moonlight-1533957233.log](https://github.com/moonlight-stream/moonlight-qt/files/2279874/Moonlight-1533957233.log)

**Comment by Sloanstar** [comment]
H.264 looks clean too.

**Comment by cgutman** [comment]
Very odd. The closest machine I have to yours is a Windows 10 laptop with Intel HD 615 graphics and it doesn't seem to display the issue. Does it always occur or is there something that seems to trigger it? Are there any other settings that make it appear or disappear?

**Comment by Sloanstar** [comment]
I'll see if I can find some other settings that trigger it, but it only started occurring since I started using -qt. Never had the effect on Chrome.

**Comment by Sloanstar** [comment]
Looks like Hardware HEVC only. I tried installing the [HEVC codec from the windows store](https://www.microsoft.com/en-us/p/hevc-video-extensions-from-device-manufacturer/9n4wgh0z6vhq) and a reboot, no change.

**Comment by cgutman** [comment]
I wonder if it's a GPU driver bug. Moonlight Chrome never could support HEVC, so you didn't see the bug there.

Can you screenshot the Driver tab in Device Manager for your client PC's GPU? I can try to find it and install it on my laptop to try to reproduce the issue.

Is your client machine also running Windows 10 1803?

Does lowering the bitrate to 20 Mbps change anything?

**Comment by Sloanstar** [comment]
![2018-08-13 22_08_00-device manager](https://user-images.githubusercontent.com/16750081/44067961-e34cb960-9f45-11e8-877d-82249d83a6c4.png)
![2018-08-13 22_08_37-device manager](https://user-images.githubusercontent.com/16750081/44067969-e7318d9e-9f45-11e8-8c4d-ec43968af5bf.png)

**Comment by Sloanstar** [comment]
Yes, client machine is also Windows 10 1803.
No change at 20 Mbps.

**Comment by Sloanstar** [comment]
I have an older Asus laptop I can try on if you think it is a client/driver issue but I dont think the windows versions are the same, so there might be too much introduction of change there to be a viable test.

**Comment by Sloanstar** [comment]
I have a "fix" and it's an issue with the laptop and Intel/Nvidia GPU selection, not anything in your code per se, unless there's a way to force select between the rendering GPU at launch and -qt isn't doing that. 

In the mean time, configuring the GF Control Panel to force the NV GPU for the executable addresses the blending.

If you give me Wiki access I can write it up with screenshots, or if you prefer it here in the issue I can do that too, let me know.

**Comment by Sloanstar** [comment]
Video Issues

Devices with Multiple Video Cards
Open the Control Panel for your video card. (nVidia shown – AMD assumed similar)

Option 1 – Globally change the preferred video card.
Pro: This should cover future application automatically.
Con: You may use more battery on applications that don’t require discrete GPU.
![image](https://user-images.githubusercontent.com/16750081/44238662-ac66b500-a183-11e8-9484-a968dba8d4ce.png)

Option 2 – Specify GPU for Moonlight Executable.
1.	Select manage 3d Settings
2.	Add the Moonlight application (Does not normally appear in the list)
![image](https://user-images.githubusercontent.com/16750081/44238680-cc967400-a183-11e8-98af-bd0a105b6c62.png)
![image](https://user-images.githubusercontent.com/16750081/44238701-dfa94400-a183-11e8-8ce7-d36ed82fd0cc.png)
![image](https://user-images.githubusercontent.com/16750081/44238714-ed5ec980-a183-11e8-8722-e745a93b8199.png)
 
Click Apply & Enjoy.

**Comment by rogeriskira** [comment]
Unfortunately on the laptop that I am using I don't have a graphics card. 
i5-7200U 
Intel HD 620
8GB ram
I realize that I have the same fix by using software decoding with HEVC. Should I also provide my log?

**Comment by cgutman** [comment]
Yeah, attach your log from the %TEMP% folder if you can. Additionally, a screenshot of your Intel GPU's properties page in Device Manager (like the one above) would be greatly appreciated.

To work around the bug, set the codec option to Force H.264

**Comment by Sloanstar** [comment]
Make sure the Intel gfx drivers are up to date as well.

**Comment by rogeriskira** [comment]
[Moonlight-1536778620.log](https://github.com/moonlight-stream/moonlight-qt/files/2376509/Moonlight-1536778620.log)
![intel](https://user-images.githubusercontent.com/42848313/45447090-970a7b00-b69c-11e8-9fba-6dc2007324c0.PNG)
I added the log file in which I used Force Hardware decoding, and Force HEVC. But as you mentioned force h.264 and Automatic do the trick. It would be nice if I could use HEVC to save on some bandwidth.

**Comment by cgutman** [comment]
Fix released in v0.5.0 (Beta 2)

**Comment by rogeriskira** [comment]
Thank so much! I can confirm it works as intended now.
